llvm/test/MC/RISCV/align-non-executable.s

```diff
@@ -0,0 +1,29 @@
+# RUN: llvm-mc --filetype=obj --triple=riscv64 --mattr=+relax %s \
```

Comment:
Add a file-level comment. You may even link to https://github.com/llvm/llvm-project/pull/76552 to help understanding.